### PR TITLE
Inital readme

### PR DESCRIPTION
**Pull request by emilbayes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+```bash
+git clone git@github.com:denmark-io/datasets.wiki.git
+```
+
+Related work
+------------
+
+Similar sites exist but are mostly raw directories of datasets.
+Some of the are:
```
Comment (AndreasMadsen) [inline README.md]: s/the/them ?

Comment (emilbayes) [inline README.md]: You are right

**Comment by emilbayes** [comment]
Once you approve I'll squash and rebase `master` :)

**Comment by AndreasMadsen** [comment]
I approve. But rebase on master? Nothing have changed, since you branched out.

**Comment by emilbayes** [comment]
Meant merge. Anyway, done!

**Pull request closed without merging**